js-api-spec/node-package-importer.node.test.ts

```diff
@@ -0,0 +1,681 @@
+    scripts: {
+      test: 'echo "Error: no test specified" && exit 1',
+    },
+    author: 'Package Author',
+    license: 'ISC',
+    ...overrides,
+  });
+}
+
+function exportsBuilder(key: string) {
+  return {
+    exports: {
+      '.': {
+        [key]: './src/sass/_styles.scss',
+      },
+      './colors/index.scss': {
+        [key]: './src/sass/colors/index.scss',
+      },
+      './_variables.scss': {
+        [key]: './src/sass/_variables.scss',
+      },
+    },
+  };
+}
+
+describe('Node Package Importer', () => {
+  describe('resolves conditional exports', () => {
+    ['sass', 'style', 'default'].forEach(key => {
+      it(`${key} at root`, () =>
+        sandbox(dir => {
+          dir.write({
+            'node_modules/foo/src/sass/_styles.scss': 'a {b: c}',
+            'node_modules/foo/package.json': manifestBuilder(
+              exportsBuilder(key)
+            ),
+          });
+          dir.chdir(
+            () => {
+              const result = compileString('@use "pkg:foo";', {
+                importers: [nodePackageImporter],
+              });
+              expect(result.css).toEqualIgnoringWhitespace('a {b: c;}');
+            },
+            {changeEntryPoint: 'index.js'}
+          );
+        }));
+      it(`${key} at root without non-pathed exports`, () =>
+        sandbox(dir => {
+          dir.write({
+            'node_modules/foo/src/sass/_styles.scss': 'a {b: c}',
+            'node_modules/foo/package.json': manifestBuilder({
+              exports: {
+                [key]: './src/sass/_styles.scss',
+              },
+            }),
+          });
+          dir.chdir(
+            () => {
+              const result = compileString('@use "pkg:foo";', {
+                importers: [nodePackageImporter],
+              });
+              expect(result.css).toEqualIgnoringWhitespace('a {b: c;}');
+            },
+            {changeEntryPoint: 'index.js'}
+          );
+        }));
+
+      it(`${key} with subpath`, () =>
+        sandbox(dir => {
+          dir.write({
+            'node_modules/foo/src/sass/_styles.scss': 'd {e: f}',
+            'node_modules/foo/src/sass/_variables.scss': 'a {b: c}',
+            'node_modules/foo/package.json': manifestBuilder(
+              exportsBuilder(key)
+            ),
+          });
+          dir.chdir(
+            () => {
+              const result = compileString('@use "pkg:foo/variables";', {
+                importers: [nodePackageImporter],
+              });
+              expect(result.css).toEqualIgnoringWhitespace('a {b: c;}');
+            },
+            {changeEntryPoint: 'index.js'}
+          );
+        }));
+      it(`${key} with index`, () =>
+        sandbox(dir => {
+          dir.write({
+            'node_modules/foo/src/sass/_styles.scss': 'd {e: f}',
+            'node_modules/foo/src/sass/_variables.scss': 'g {h: i}',
+            'node_modules/foo/src/sass/colors/index.scss': 'a {b: c}',
```

Comment:
Same issue as above with "colors" here. Maybe `subdir` instead to communicate the aspect of the difference that's relevant to this test.